adafruit_is31fl3741/__init__.py

```diff
@@ -233,6 +234,18 @@ def show(self) -> None:
                 i2c.write(self._pixel_buffer, start=180, end=352)
                 self._pixel_buffer[180] = save
 
+    def write(self, mapping: Tuple, buffer: ReadableBuffer) -> None:
+        """
+        Write buf out on the I2C bus to the IS31FL3741.
+
+        :param mapping: map the pixels in the buffer to the order addressed by the driver chip
+        :param buffer: The bytes to clock out. No assumption is made about color order
+        :return: None
+        """
+        for pos, data in enumerate(buffer):
+            self[mapping[pos]] = data
```

Comment:
```suggestion
            if mapping[pos] != 65535:
                self[mapping[pos]] = data
```

The mappings for the glasses use 65535 as an "invalid" number since some pixels do not exist.